website/docs/admin_hosts_maya.md

```diff
@@ -107,6 +107,18 @@ or Deadlines **Draft Tile Assembler**.
 This is useful to fix some specific renderer glitches and advanced hacking of Maya Scene files. `Patch name` is label for patch for easier orientation.
 `Patch regex` is regex used to find line in file, after `Patch line` string is inserted. Note that you need to add line ending.
 
+## Load Plugins
+
+### Reference Loader > Namespace
+Here you can create your own custom naming for the reference loader. <br>
+The custom naming is split into two parts separated by a ":" symbol. The first half is the namespace and the second half is the group name of the reference. If you don't set the namespace or the group name (or both), the default namespace and group name will be applied.
```

Comment:
So if I understand correctly, you want this field to be filled by default with the originally generated namespace, and not blank ?

Reply:
Yes, the result will be the same but then it could _always_ use the value from settings. Seems more explicit to me. No? I wonder what the OP crew thinks however.

Reply:
Yes I agree with you, it could even be simpler. Do I wait for feedback from OP crew or do I make the modification now ?

Reply:
@iLLiCiTiT @mkolar any thoughts on this by any chance? (not sure who best to tag, sorry)